src/Dapr.AspNetCore/FromStateAttribute.cs

```diff
@@ -18,19 +18,28 @@ public class FromStateAttribute : Attribute, IBindingSourceMetadata
         /// <summary>
         /// Initializes a new instance of the <see cref="FromStateAttribute"/> class.
         /// </summary>
-        public FromStateAttribute()
+        /// <param name="stateStoreName">The state store name.</param>
+        public FromStateAttribute(string stateStoreName)
         {
+            this.StateStoreName = stateStoreName;
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="FromStateAttribute"/> class.
         /// </summary>
+        /// <param name="stateStoreName">The state store name.</param>
         /// <param name="key">The state key.</param>
-        public FromStateAttribute(string key)
+        public FromStateAttribute(string stateStoreName, string key)
         {
+            this.StateStoreName = stateStoreName;
             this.Key = key;
         }
 
+        /// <summary>
+        /// Gets the state store name.
+        /// </summary>
+        public string StateStoreName { get; }
```

Comment:
nit: you used `StoreName` elsewhere, I think `StoreName` is better, a little more brief.